### PR TITLE
Add muteRoom to Dance Party; fix muteRoom.

### PR DESCRIPTION
**Pull request by seangibat**

### CODE REVIEW

**Comment by hndrewaall** [comment]
Why is there a capacity to the dance party at all? Shouldn’t it be unlimited?

**Comment by hndrewaall** [comment]
test

**Comment by seangibat** [comment]
idk, to make it a scarcer resource or something? i can remove it

**Comment by hndrewaall** [comment]
i think we should remove it, but i’ll let @morganecf chime in

**Comment by morganecf** [comment]
@hndrewaall the capacities are completely inactive right now. I just added them to the room config when I first made the config and never removed them, but I'll clean all that up when I work on enforcing room capacities for certain rooms like closets/bathrooms.

File: app/client/jsx/JitsiVideo.jsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ class JitsiVideo extends Component {
                     SHOW_JITSI_WATERMARK: false,
                     DEFAULT_REMOTE_DISPLAY_NAME: 'Fellow Clarendonite',
                     SHOW_WATERMARK_FOR_GUESTS: false,
-                    TOOLBAR_BUTTONS: this.toolbarButtons,
+                    TOOLBAR_BUTTONS: toolbarButtons,
```
Comment (morganecf) [inline app/client/jsx/JitsiVideo.jsx:76]: did you remove this in previous PR? oops

Comment (seangibat) [reply app/client/jsx/JitsiVideo.jsx:76]: hm no, I think you may be readin the PR wrong

Comment (seangibat) [reply app/client/jsx/JitsiVideo.jsx:76]: hm no, I think you may be readin the PR wrong

**Pull request merged**